subseas_flex_fcst/maproom.py

```diff
@@ -396,21 +396,6 @@ def fcst_tile_url_callback(proba, variable, percentile, threshold, startDate, le
 def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold, startDate,leadTime):
     # Reading
     fcst_mu, fcst_var, dofVar, obs, hcst = read_cptdataset(leadTime, startDate, y_transform=CONFIG["y_transform"])
-    # Get Issue date and Target season
```

Comment:
I probably meant to print a title for the map and somehow failed to do so... we can take care of that later.

Reply:
Made into an issue for later.